agent/config/config_windows.go

```diff
@@ -57,6 +57,8 @@ func DefaultConfig() Config {
 	programData := utils.DefaultIfBlank(os.Getenv("ProgramData"), `C:\ProgramData`)
 	ecsRoot := filepath.Join(programData, "Amazon", "ECS")
 	dataDir := filepath.Join(ecsRoot, "data")
+	DefaultPauseContainerImageName = "amazonaws.com/ecs/pause-windows"
+	DefaultPauseContainerTag = "latest"
```

Comment:
Considering that ECS would support future releases of Windows, it would be advisable to use a generic tag. We can probably consider checking the version of windows on the pause container to ensure that it is using the supported version.